.github/workflows/on-main.yml

```diff
@@ -13,10 +13,10 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Use Node.js ${{ matrix.node-version }}
+      - name: Use Node.js 16
         uses: actions/setup-node@v1
```

Comment:
Is anything blocking to update to `actions/setup-nodev3` instead? It uses `Node 16` by default as per the [docs](https://github.com/actions/setup-node/releases/tag/v3.0.0).

Reply:
Oh nice, I wasn't aware of that! I'll give it a try.

Reply:
Updating to v3 of that action, but I'm going to keep the `node-version` input in place based on the docs:

> However, it is recommended to always specify Node.js version and don't rely on the system one.